string name
float32 area
